MusicTaste/MusicTaste/View Controllers/Login/LoginViewController.m

```diff
@@ -0,0 +1,110 @@
+//
+//  LoginViewController.m
+//  MusicTaste
+//
+//  Created by Aman Abraham on 7/6/22.
+//
+
+#import "LoginViewController.h"
+#import <Parse/Parse.h>
+#import "ConnectView.h"
+#import "LoginViewController.h"
+
+static NSString * const SpotifyClientID = @"9f9a8a428178497e8c58840c65d9f3c0";
+static NSString * const SpotifySecretID = @"085725985e87480fab42de4970086a54";
+static NSString * const SpotifyRedirectURLString = @"spotify-ios-quick-start://spotify-login-callback";
+
+
+@interface UIViewController ()
+
+@property (nonatomic, assign) BOOL didAuthorize;
+
+@end
+
+@implementation LoginViewController
+
+@synthesize didAuthorize = _didAuthorize;
+
+#pragma mark - Authorization example
+
+- (void)viewDidLoad
+{
+    SPTConfiguration *configuration = [SPTConfiguration configurationWithClientID:SpotifyClientID
+                                                                      redirectURL:[NSURL URLWithString:SpotifyRedirectURLString]];
+    self.sessionManager = [SPTSessionManager sessionManagerWithConfiguration:configuration
+                                                                       delegate:self];
+    [super viewDidLoad];
+    if(self.didAuthorize){
+    }
+    else{
+    }
+}
+
+#pragma mark - Actions
+
+- (void)didTapAuthButton:(ConnectButton *)sender
+{
+    SPTScope scope = SPTUserLibraryReadScope | SPTPlaylistReadPrivateScope;
+    if (@available(iOS 11, *)) {
+        [self.sessionManager initiateSessionWithScope:scope options:SPTDefaultAuthorizationOption ];
+    } else {
+        [self.sessionManager initiateSessionWithScope:scope options:SPTDefaultAuthorizationOption presentingViewController:self];
+    }
```

Comment:
Are you planning to support iOS versions older than iOS 11? If not then you don't need an if statement here